microprofile/galleon-common/src/main/resources/layers/standalone/microprofile-rest-client/layer-spec.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" ?>
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2020, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="microprofile-rest-client">
+    <packages>
+        <package name="org.jboss.resteasy.resteasy-client-microprofile" optional="true"/>
```

Comment:
Is optional="true" needed? If someone asks for this layer there's no reason to exclude the package.